migrations/sql/R__stored_procedure_for_mine_information.sql

```diff
@@ -532,8 +556,18 @@ CREATE OR REPLACE FUNCTION transfer_mine_information() RETURNS void AS $$
                 now()
             FROM new_record new;
             SELECT count(*) FROM mine_location into new_row;
+
+            -- Remove invalid rows that were updated or inserted
+            WITH deleted_rows AS (
+                DELETE FROM mine_location
+                WHERE latitude IS NULL OR longitude IS NULL
+                RETURNING 1
+            )
+            SELECT COUNT(*) FROM deleted_rows INTO delete_row;
```

Comment:
Backwards-compatibility: The API-level validation enforces that a `mine_location` record must have both a `latitude` and `longitude` value. If either value is missing, the POST is rejected. If neither value is provided, the record is not created.

To stay consistent with this behaviour, I am removing any records that were created with `null` lat and/or long values or were updated to include a `null` lat/long value.

Reply:
Hmm 🤔 Interesting point on enforcing API rules at the ETL level as well.
Currently if I run 
```
SELECT FROM mine_location
WHERE latitude IS NULL OR longitude IS NULL
```
I dont get anything but I dont see this causing any harm either

Reply:
Explained IRL.

For future reference:
This is consistency with our apps expectations, not with existing data. The app can't handle `null` lat/long values. The lat & long must both be present or the `mine_location` record is not created.